pkg/rtc/participant.go

```diff
@@ -434,7 +451,41 @@ func (p *ParticipantImpl) Close() error {
 }
 
 func (p *ParticipantImpl) Negotiate() {
-	p.subscriber.Negotiate()
+	if p.MigrateState() != types.MigrateStateInit {
+		p.subscriber.Negotiate()
+	}
+}
+
+func (p *ParticipantImpl) InitSubscribePreviousAnwser(previous *webrtc.SessionDescription) {
+	p.subscriber.SetInitPreviousAnwser(previous)
```

Comment:
The `SetInitPrevious` is confusing me. Did you mean to say `Initial previous answer`. Can we just call it `SetPreviousAnswer`?

Reply:
Also `Answer` spelling is incorrect.

Reply:
Renamed